modules/homer/pos2bed/main.nf

```diff
@@ -0,0 +1,33 @@
+def VERSION = '4.11' // Version information not provided by tool on CLI
+
+process HOMER_POS2BED {
+    tag "$meta.id"
+    label 'process_medium'
+
+    conda (params.enable_conda ? "bioconda::homer=4.11=pl526hc9558a2_3" : null)
```

Comment:
I think there was an issue when using the build numbers for conda, so they have not been used anymore

Reply:
So this?
```suggestion
    conda (params.enable_conda ? "bioconda::homer=4.11" : null)
```

Reply:
yes